geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -185,10 +202,11 @@ public LocatorClusterManagementService(InternalConfigurationPersistenceService p
     }
 
     // persist configuration in cache config
-
-    configurationManager.add(config, groupName);
+    for (String groupName : groups) {
+      configurationManager.add(config, groupName);
+    }
     result.setStatus(StatusCode.OK,
-        "Successfully updated configuration for " + groupName + ".");
+        "Successfully updated configuration for " + StringUtils.join(groups, ", ") + ".");
```

Comment:
Prefer `String.join()`, which is built into Java.

Reply:
done